tests/test_utils.py

```diff
@@ -0,0 +1,28 @@
+from cf_remote.utils import parse_envfile
+
+
+def test_parse_envfile():
+    data = parse_envfile("NTD_TEST=\"test\"")
+    assert "NTD_TEST" in data 
+    assert data["NTD_TEST"] == "test"
+
+    data = parse_envfile("NTD_X\"test\"")
+    assert not data
+
+    data = parse_envfile("=\"test\"")
+    assert not data
+
+    data = parse_envfile("NTD_test=\"test\"")
+    assert not data
+
+    data = parse_envfile("NTD_TEST=\"hello\"world\"")
+    assert "\"" in data["NTD_TEST"]
```

Comment:
If this should be allowed (an unescaped double quote) it should result in:

```suggestion
    data = parse_envfile("NTD_TEST=\"hello\"world\"")
    assert data["NTD_TEST"] == 'hello"world'
```

Or we could decide to make it an error (since it seems the sender did not escape the string properly.